File: DrinkItUpWebApp/DrinkItUpWebApp/Models/UserModel.cs
using Microsoft.AspNetCore.Mvc;
using System.ComponentModel.DataAnnotations;

namespace DrinkItUpWebApp.Models
{
    public class UserModel
    {
        [Required]
        [MaxLength(70)]
        public string Email { get; set; }
        [Required]
        [MinLength(6)]
        public string Password { get; set; }
        [Required]
        [MaxLength(70)]
        public string UserNameToShow { get; set; }
        [Required]
        [DataType(DataType.Date)]
        public DateTime DateOfBirth { get; set; }
        

    }
}

File: DrinkItUpWebApp/DrinkItUpSPA/src/app/shared/user.model.ts
export class UserModel{
    userId?: number;
    userNameToShow?: string;
    email?: string;
    dateOfBirth?: Date;
    token?: string;
}

export class UserRegisterModel{
    constructor(
    public userNameToShow: string,
    public email: string,
    public password: string,
    public dateOfBirth: Date){}

}